### DISCUSSION TITLE
Where to find the Intrinsics & Extrinsics of the robot camera?

### DISCUSSION THREAD

**Discussion by merak0514** [cyberbotics/wrestling#54] [Q&A] [answered]
Hello, do you know where can we find the intrinsic and extrinsic camera parameters? It is really helpful to compute the distances.

Many thanks!

**Comment by omichel** [comment, marked as answer]
The camera parameters can be found in the Nao proto:
- [Field of view](https://github.com/cyberbotics/wrestling/blob/main/protos/Nao.proto#L98) is 1.064 rad.
- [Default values](https://github.com/cyberbotics/wrestling/blob/main/protos/Nao.proto#L26-L31)  of the Nao cameras, including the resolution which is 160x120 pixels.
- Implementation of the [Nao cameras](https://github.com/cyberbotics/wrestling/blob/main/protos/Nao.proto#L824-L855).

The documentation for the Webots Camera node is available [here](https://cyberbotics.com/doc/reference/camera).

Please let me know if you have any question regarding the camera parameters.
